src/Turing.jl

```diff
@@ -28,6 +29,9 @@ Enable progress logging in Turing if `progress` is `true`, and disable it otherw
 function setprogress!(progress::Bool)
     @info "[Turing]: progress logging is $(progress ? "enabled" : "disabled") globally"
     PROGRESS[] = progress
+    Logging.with_logger(Logging.NullLogger()) do
+        AbstractMCMC.setprogress!(progress)
```

Comment:
Should we just add a `quiet` or `silent` keyword argument to `AbstractMCMC.setprogress!` (and possibly the other `setprogress!` functions as well).